Fact-checking and copyediting throughout

pauamma opened the pull request:
Should "$FreeBSD$" still be there, since this is now a port?

pauamma committed:
Fact-checking and copyediting throughout

pauamma commented:
ping

pauamma commented:
Are you (the repo committer(s)) waiting for something from me?

pauamma commented:
Ping @stesser @lsalvadore

lsalvadore commented:
I am sorry, watching the repository was disabled and I only saw all open pull requests now that you mentioned me. Now I am watching the repository, so this should not happen again.

I will review this pull request and all the others open pull requests (#5, #6 , #7, #8) as soon as possible.

pauamma commented:
> I will review this pull request and all the others open pull requests (#5, #6 , #7, #8) as soon as possible.

Thanks.

pauamma commented:
*p/ing*

lsalvadore commented:
I am sorry this is taking long, but as far as  I know I am the only more or less active committer on this repository and I have many other priorities too, both inside and outside FreeBSD. I will do my best to be faster on calenda-data.

Before merging other pull requests, I would like to fix the fact that no license is included in the repository, but the task is not an easy one. If you want to give a hand, please check pull request #9.

While that pull request goes forward, I am going to at least review and approve (or ask changes) for existing pull requests, so that as soon as the license issue is fixed all the pull requests can be merged.

pauamma commented:
> Before merging other pull requests, I would like to fix the fact that no license is included in the repository, but the task is not an easy one. If you want to give a hand, please check pull request #9.

Commented there. 

> While that pull request goes forward, I am going to at least review and approve (or ask changes) for existing pull requests, so that as soon as the license issue is fixed all the pull requests can be merged.

*nod*

pauamma commented:
*ping*

lsalvadore commented on calendar.history:
```diff
@@ -420,12 +425,12 @@
 08/10	Chicago incorporated as a village of 300 people, 1833
 08/10	Major terrorist plot to destroy aircraft disrupted by
-	Scotland Yard, 2006
+	Scotland Yard, 2006*
```
Why are you adding a `*` here? Is it a typo or am I missing something?

pauamma replied:
> Why are you adding a `*` here? Is it a typo or am I missing something?
Typo. Possibly an instance of my malfunctioning keyboard that I missed. Will fix.

lsalvadore commented on calendar.history:
```diff
@@ -501,7 +507,7 @@
 09/13	British defeat the French at the Plains of Abraham, just outside the walls of Quebec City, 1759
 09/13	Building of Hadrian's Wall begun, 122
-09/13	Chiang Kai-Shek becomes president of China, 1943
+09/13	Chiang Kai-shek becomes president of China, 1943
```
There are many ways to transliterate Chinese names. I suggest using pinyin and putting in parentheses other transliterations when they are well known (in this case, according to wikipedia, the one you used should be a mixture of Wade-Giles and Cantonese: https://en.wikipedia.org/wiki/Chiang_Kai-shek#Names).

pauamma replied:
No objection to that. Here, I was sticking to removing the spurious uppercase letter.

lsalvadore commented on calendar.history:
```diff
@@ -587,34 +593,34 @@
 10/18	The Grand Ole Opry opens its doors, 1925
 10/19	Boston Shoemakers form first American labor organization, 1648
-10/19	Mao Tse-tung establishes the People's Republic of China, 1949
+10/19	Mao Zedong establishes the People's Republic of China, 1949
```
As above, I suggest using pinyin transliteration and putting in parentheses other well known transliterations. Thus, I suggest "Mao Zedong (Wade-Giles: Mao Tse-tung)". See also https://en.wikipedia.org/wiki/Mao_Zedong#English_romanisation_of_name .

pauamma replied:
WFM.

lsalvadore commented on calendar.history:
```diff
@@ -587,34 +593,34 @@
 10/27	Stock markets around the world crash because of fears of
 	a global economic meltdown, 1997
-10/28	Columbus discovers Cuba, 1492
+10/28	Columbus gets to Cuba about 4600 years too late to discover it, 1492
```
I guess that you are trying to make this entry less Eurocentric, but I do not think irony is the solution here. Moreover, the entry is a bit unclear: who discovered Cuba 4600 years earlier?  Are you referring to some native pre-Columbian population?

I like more what you did with Captain Cook on 01/17. I would then suggest to switch it to "Columbus is the first European explorer to discover Cuba, 1492".

pauamma replied:
https://en.wikipedia.org/wiki/History_of_Cuba#Pre-Columbian_(to_1500)

How about "Colombus reaches Cuba about 4600 years after its first settlers did, 1492" (trying to avoid "discover")

lsalvadore replied:
I am fine with avoiding the word "discover" but I would like to avoid moving the focus from Columbus to the native populations: the calendar entry is about what Columbus did, not about who did the same thing before him.

I suggest "Columbus lands in Cuba" or, if you really want to state that someone was already there, "Columbus lands in Cuba: it is the first meeting between Europeans and Cuban natives".

pauamma replied:
> I am fine with avoiding the word "discover" but I would like to avoid moving the focus from Columbus to the native populations: the calendar entry is about what Columbus did, not about who did the same thing before him.
> 
> I suggest "Columbus lands in Cuba" or, if you really want to state that someone was already there, "Columbus lands in Cuba: it is the first meeting between Europeans and Cuban natives".

The latter works for me.

bsdimp replied:
I think "Columbus lands in Cuba" is a fine balance. It's brief, it avoids saying he discovered it, if avoids implying that this is when the rampant colonization of the new world began, suppressing the native populations to the imperialist whims of Spain, and later others. Also, any other amplification of the first European may or may not be entirely accurate. We know the vikings were in the new world 400 years before. We know they sailed across at least part of the Atlantic by at least a hundred years before Columbus. etc. Keeping it simple, and generic gives the right information and right weight to it w/o getting bogged down in all these details. Plus, the data of first settlement is constantly changing, and 4600 years may well wind up being a late date in the future as the archeological record expands.

pauamma replied:
Going for "Columbus gets to Cuba and meets inhabitants". Compare with "10/12	Bahama Natives discover Columbus of Europe lost on their shores" and "05/04	Christopher Columbus sets foot on Jamaica" for a range of wordings of similar events.

lsalvadore reviewed (commented):
Thanks @pauamma and sorry for the delay.

The changes are a bit difficult to review: they are many and of different kinds. So I am trusting you on most of the things and make only a few comments.

In the future please open more pull requests with changes focused on a single topic, so I can focus on the single topics and review them one at the time. For example in this case you could have opened a pull request for Fahrenheit/Celsius conversion, another one on Chinese History, another one on explorers, another one on US History etc. Don't be afraid to open too many pull requests: I rarely find the time to make long and complex reviews, but I can easily find the time for small reviews. So I would be able to merge your changes much quickier.

pauamma committed:
Address review comments.

Also correct Lizzie Borden's spelling and the date of her acquittal (per Wikipedia).

lsalvadore commented on calendar.history:
```diff
@@ -501,7 +507,8 @@
 09/13	British defeat the French at the Plains of Abraham, just outside the walls of Quebec City, 1759
 09/13	Building of Hadrian's Wall begun, 122
-09/13	Chiang Kai-Shek becomes president of China, 1943
+09/13	Jiǎng Jièshí (Wade-Giles: Chiang3 Chieh4-shih2, known in the West as Chiang Kai-shek) becomes
```
I think 3, 4 and 2 should be superscript and that UTF-8 supports such superscripts, isn't it? If it is, please switch those digits to superscripts.

pauamma replied:
I learned something today. I thought you had to use HTML to get those.

pauamma committed:
Use Unicode superscript digits
